gaia-list.js

```diff
@@ -256,15 +256,6 @@ module.exports = component.register('gaia-list', {
       width: auto;
     }
 
-    /**
-     * Reverse the icons when the document is RTL mode
-     */
-
-    :host-context([dir=rtl]) ::content i:before {
-      transform: scale(-1, 1);
-      text-align: end;
-    }
-
```

Comment:
MMhh actually, in this case, why not putting rtl rules in gaia-icons directly, as now `forward`, `back`, and their light counterparts icons does not exists any more in gaia-fonts?